core/src/io/anuke/mindustry/world/blocks/distribution/MassDriver.java

```diff
@@ -181,6 +181,11 @@ public void drawConfigure(Tile tile){
 
         MassDriverEntity entity = tile.entity();
 
+        entity.waitingShooters.forEach(shooter -> {
+            Drawf.circles(shooter.drawx(), shooter.drawy(), (tile.block().size / 2f + 1) * tilesize + sin - 2f, Pal.place);
+            Drawf.arrow(shooter.drawx(), shooter.drawy(), tile.drawx(), tile.drawy(), size * tilesize + sin, 4f + sin);
```

Comment:
This should be colored `Pal.place` to match the selection circle. This would require making `Drawf#arrow` have an extra color parameter, `color`, and making a new overload of `Drawf#arrow` that calls the new `Drawf#arrow` method with `Pal.accent` as the default color.

Reply:
Noted, did cross my mind but i was unsure to the reasoning that the arrow color was hardcoded so hence the draft pull for comments 👍